cmake/switch/Switch.cmake

```diff
@@ -14,8 +14,9 @@ set(CMAKE_ASM_FLAGS_INIT "${NX_COMMON_FLAGS}")
 
 set(CMAKE_EXE_LINKER_FLAGS_INIT "${NX_ARCH_SETTINGS} ${NX_LIB_DIRS} -fPIE -specs=${DEVKITPRO}/libnx/switch.specs")
 
-set(CMAKE_FIND_ROOT_PATH
-	${CMAKE_FIND_ROOT_PATH}
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/switch")
```

Comment:
This is the only thing I'm not so happy about because [the documentation of `CMAKE_MODULE_PATH`](https://cmake.org/cmake/help/latest/variable/CMAKE_MODULE_PATH.html) says that this variable is intented for projects.